NitroxClient/Communication/Packets/Processors/CyclopsChangeNameProcessor.cs

```diff
@@ -0,0 +1,45 @@
+﻿using NitroxClient.Communication.Packets.Processors.Abstract;
+using NitroxClient.GameLogic.Helper;
+using NitroxModel.DataStructures.Util;
+using NitroxModel.Helper;
+using NitroxModel.Packets;
+using System;
+using UnityEngine;
+
+namespace NitroxClient.Communication.Packets.Processors
+{
+    public class CyclopsChangeNameProcessor : ClientPacketProcessor<CyclopsChangeName>
+    {
+        private PacketSender packetSender;
+
+        public CyclopsChangeNameProcessor(PacketSender packetSender)
+        {
+            this.packetSender = packetSender;
+        }
+
+        public override void Process(CyclopsChangeName namePacket)
+        {
+            Optional<GameObject> opCyclops = GuidHelper.GetObjectFrom(namePacket.Guid);
+
+            if (opCyclops.IsPresent())
+            {
+                CyclopsNameScreenProxy ScreenProxy = opCyclops.Get().GetComponentInChildren<CyclopsNameScreenProxy>();
+
+                if (ScreenProxy != null)
```

Comment:
Future reference: the current convention is to have an else to log that this was not found. This prevents us from missing a silent failure.